libs/core/src/components/pds-popover/pds-popover.tsx

```diff
@@ -0,0 +1,175 @@
+import { Component, Element, Event, Host, Prop, State, h, EventEmitter, Method } from '@stencil/core';
+import {
+  positionTooltip
+} from '../../utils/overlay';
+
+/**
+ * @slot (default) - The popover's target element
+ * @slot content - HTML content for the popover
+ */
+
+@Component({
+  tag: 'pds-popover',
+  styleUrl: 'pds-popover.scss',
+  shadow: true,
+})
+export class PdsPopover {
+  private contentEl: HTMLElement | null;
+
+  /**
+   * Reference to the Host element
+   */
+  @Element() el: HTMLPdsPopoverElement;
+
+  /**
+   * Determines when the popover is open
+   * @defaultValue false
+   */
+  @State() isOpen = false;
+
+  /**
+   * A unique identifier used for the underlying component id attribute.
+   */
+  @Prop() componentId: string;
+
+  /**
+   * Determines whether or not the popover has an arrow
+   * @defaultValue true
+   */
+  @Prop() hasArrow? = false;
+
+  /**
+   * Determines the preferred position of the popover
+   * @defaultValue "right"
+   */
+  @Prop({ reflect: true }) placement:
+    'top'
+    | 'top-start'
+    | 'top-end'
+    | 'right'
+    | 'right-start'
+    | 'right-end'
+    | 'bottom'
+    | 'bottom-start'
+    | 'bottom-end'
+    | 'left'
+    | 'left-start'
+    | 'left-end' = 'right';
+
```

Comment:
Extract to a Type

Reply:
Detail what `start` and `end` means. Do we need an Internationalization section?